src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListDataGrid.tsx

```diff
@@ -303,16 +376,17 @@ const TaskListDataGrid: React.FC<TaskListDataGridProps> = ({
   }
 
   const isCellEditable = (params: GridCellParams) => {
-    return params.id === -1;
+    return params.id === -1 || params.id === currentlyEditingId;
   };
 
   return (
     <DataGrid
       columns={columns}
       rows={rows}
+      editMode="row"
       isCellEditable={isCellEditable}
-      experimentalFeatures={{ newEditingApi: true }}
```

Comment:
I'm pretty sure you need this for processRowUpdate to work? Why did you switch to making everything custom instead of using the built in editingapi stuff